source/WrappedText.h

```diff
@@ -69,7 +76,6 @@ class WrappedText {
 	
 private:
 	void SetText(const char *it, size_t length);
-	void Wrap();
```

Comment:
Why do we need to change the visibility of the method?

Reply:
Probably it was needed at some time and I forgot to restore it.